addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10313,7 +10313,11 @@ msgctxt "#20244"
 msgid "Android Apps"
 msgstr ""
 
-#empty strings from id 20245 to 20249
+msgctxt "#20245"
```

Comment:
Well, we could also list these shortcuts in our settings system so that application and OS related settings (which they already partly are) are listed in the same place. I'm fine with giving the current VFS approach a try though.